src/sas/sascalc/invariant/invariant.py

```diff
@@ -817,10 +817,17 @@ def get_surface(self, contrast, porod_const, extrapolation=None):
         Historically, Sv was computed with the invariant and the Porod
         constant so as not to have to know the contrast in order to get the
         Sv as:
+
+        .. :math::
+
             surface = (pi * V * (1- V) * porod_const) / q_star
+
```

Comment:
Use latex markup here and elsewhere, otherwise you get, e.g., `q subscript s t a r` all italic with math spacing rather than `q^*`.

Reply:
Given the style of the document is written in python variables rather than maths variables, I've turned these into literal blocks instead (plus a few other places in the same file that had similar problems, but weren't making sphinx emit warnings)